docs/en/UI/AspNetCore/Navigation-Menu.md

```diff
@@ -100,7 +100,7 @@ There are more options of a menu item (the constructor of the `ApplicationMenuIt
 * `url` (`string`): The URL of the menu item.
 * `icon` (`string`): An icon name. Free [Font Awesome](https://fontawesome.com/) icon classes are supported out of the box. Example: `fa fa-book`. You can use any CSS font icon class as long as you include the necessary CSS files to your application.
 * `order` (`int`): The order of the menu item. Default value is `1000`. Items are sorted by the adding order unless you specify an order value.
-* `customData` (`object`): A custom object that you can associate to the menu item and use it while rendering the menu item.
+* `customData` (`Dictionary<object, string>`): A dictionary that allows storing custom objects that you can associate with the menu item and use it while rendering the menu item.
```

Comment:
```suggestion
* `customData` (`Dictionary<string, object>`): A dictionary that allows storing custom objects that you can associate with the menu item and use it while rendering the menu item.
```

Reply:
Almost missed it, thanks for your attention